server.js

```diff
@@ -14,10 +16,92 @@ const welcomeMessage = {
 //This array is our "data store".
 //We will start with one message in the array.
 //Note: messages will be lost when Glitch restarts our server.
-const messages = [welcomeMessage];
+const messages = [];
+let nextMessageID = 0;
 
 app.get("/", function (request, response) {
   response.sendFile(__dirname + "/index.html");
 });
 
-app.listen(process.env.PORT);
+app.post("/message", (req, res) => {
+  if (req.body.from !== "" && req.body.text !== "") {
+    nextMessageID++;
+    const message = {};
+    message.id = nextMessageID;
+    message.from = req.body.from;
+    message.text = req.body.text;
+    // message.timeSent = Date.now();
+    // message.timeSent = new Date().toDateString();
+    // message.timeSent = new Date().toISOString();
+    message.timeSent = new Date().toString();
```

Comment:
This will store a date with the format "Tue Aug 19 1975 23:15:30" ([link](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toString)).
It's much better to store dates in ISO format using `toISOString()` (which you have commented above), so that the timezone offset is always standard as UTC; it's much less confusing.